src/distributed_graph_generator.cc

```diff
@@ -185,7 +185,7 @@ void distributed_graph_generator::generate_distributed_commands(const task& tsk)
 	const node_id reduction_initializer_nid = 0;
 
 	const box<3> empty_box({0, 0, 0}, {0, 0, 0});
```

Comment:
```suggestion
	const box<3> empty_reduction_box({0, 0, 0}, {0, 0, 0});
```
This is used in one location, when generating notification-only reduction pushes. Should it also be renamed?